src/scenes/PlantingSitesRouter/editor/Zones.tsx

```diff
@@ -138,45 +162,74 @@ export default function Zones({ onChange, onValidate, site }: ZonesProps): JSX.E
         hasTutorial: true,
         handlePrefix: (prefix: string) => strings.formatString(prefix, <MapIcon icon='slice' />) as JSX.Element[],
       },
+      {
+        text: strings.SITE_ZONE_BOUNDARIES_SIZE,
+        isBold: true,
+      },
     ],
     []
   );
 
-  const onEditableBoundaryChanged = (featureCollection?: FeatureCollection) => {
+  const onEditableBoundaryChanged = (editableBoundary?: FeatureCollection) => {
```

Comment:
This function is getting a bit beefy, maybe we should split it out so we can add unit tests?

Reply:
Thanks for the suggestion. Refactored into a utility which can now be reused for the next effort - subzone size validation. Added tests, etc.